### ISSUE TITLE
Observable.Add(handler) to a ISignal doesn't execute the handle function when the property value has changed

### ISSUE THREAD

**Issue by Lleutch** [ReedCopsey/Gjallarhorn#32]
As the title suggest, when trying to `Observable.Add( handler )` there is an issue where, when changing the property binded to this Observable, the handler is not called.

This issue has been reproduced in the FSI by referencing only Gjallarhorn.dll.
The steps are the following :

run the following code line by line by sending it to F# interactive.
The last 3 lines, do not trigger any of the two observables handlers.

```open Gjallarhorn

module test =
    type V = { A : int ; B : int }

    let v = Mutable.create { A = 0 ; B = 0 } 

    let a = v |> Signal.map (fun v -> v.A) 
    let b = v |> Signal.map (fun v -> v.B) 

    a |> Observable.add (fun a -> printfn "A changed to %d" a) 
    b |> Observable.add (fun b -> printfn "B changed to %d" b) 

    v.Value <- { v.Value with A = 12 } 
    v.Value <- { v.Value with B = 2 }
    v.Value <- {A = 5 ; B = 6} ```

**Comment by ReedCopsey** [comment]
Thank you for the report.  I'll take a look at see if I can figure out why this is happening.

Note that using `Signal.Subscription.create` instead of `Observable.add` is functioning properly - there must be something wrong with the propagation of change events for observables.

**Comment by FoggyFinder** [comment]
Not sure I understand what the problem is:

![1](https://cloud.githubusercontent.com/assets/16510738/22482436/a8ec49ea-e801-11e6-9f6d-945b7d7d0f53.jpg)

**Comment by ReedCopsey** [comment]
@FoggyFinder If you only send the first portion, and don't send the last 3 lines, then send them one at a time, you don't get the notification...

There's an issue with notification of the observables happening somewhere in there.

**Comment by FoggyFinder** [comment]
thanks, got it - I did not pay attention to the "line by line" =(

**Comment by ReedCopsey** [comment]
Fixed in 0.0.9 - now on NuGet https://www.nuget.org/packages/Gjallarhorn.Bindable.Wpf/

Let me know if you have any other issues.
